core/src/test/java/org/apache/iceberg/TestFastAppend.java

```diff
@@ -317,6 +317,60 @@ public void testRecoveryWithoutManifestList() {
     assertThat(metadata.currentSnapshot().allManifests(FILE_IO)).contains(newManifest);
   }
 
+  @TestTemplate
+  public void testWriteNewManifestsIdempotency() {
+    table.updateProperties().set(TableProperties.MANIFEST_LISTS_ENABLED, "true").commit();
```

Comment:
This property doesn't do anything anymore right? I saw some tests using it in this class so I kept it around